### PR TITLE
Fixing issue when sending 0 as integer in dtmf.

### PR DESCRIPTION
**Pull request by PerezArmando**
I was getting the following error when sending 0 as an integer to the session's dtmf function.

image

Steps to reproduce:
1. Open a voice session, and call session.dtmf(0).
2. Open the browser's console and lookup for the error.

Tested on:
1. Google Chrome Version 78.0.3904.97 (Official Build) (64-bit)
2. Angular 8.2.7
3. SIP.js 0.14.0 (even though, the up to date version contains the same issue).

### CODE REVIEW

**Comment by seanbright** [comment]
There is similar code [here](https://github.com/onsip/SIP.js/blob/388d1b84e7e806d89ece3a76175b9e41890ad048/src/Session/DTMF.ts#L39-L53):

https://github.com/onsip/SIP.js/blob/388d1b84e7e806d89ece3a76175b9e41890ad048/src/Session/DTMF.ts#L39-L53

Maybe this should be extracted out somehow so it could be used from both places?

**Comment by PerezArmando** [comment]
Thanks for pointing out that @seanbright. I will work on that and update the PR.

File: src/Session/DTMFValidator.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    // Check tone type
+    if (typeof tone === "string" ) {
+      tone = tone.toUpperCase();
+    } else if (typeof tone === "number") {
+      tone = tone.toString();
+    } else {
+      throw new TypeError("Invalid tone: " + tone);
+    }
+    // Check tone value
+    if (!tone.match(/^[0-9A-D#*]$/)) {
```
Comment (PerezArmando) [inline src/Session/DTMFValidator.ts]: Show we allow the comma here?

**Comment by seanbright** [comment]
IMO, yes, but I am not one of the project maintainers so I can't say if that is acceptable. It does seem to be supported by [`RTCDTMFSender.insertDTMF()`](https://developer.mozilla.org/en-US/docs/Web/API/RTCDTMFSender/insertDTMF).

**Comment by PerezArmando** [comment]
@seanbright, I ran into some problems running some npm scripts on my windows machine, so I had to commit in order to check every time that I did not break any tests. Are you working on Windows?

**Comment by egreenmachine** [comment]
This looks great and I am going to take it. Just be aware that this is all on our "legacy" API and will not be taken forward in our 0.16 release.

**Pull request merged**